app/utils/Utils.scala

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
Please add test class for this as values like "" or  " " are very easy to miss (in PR reviews) if someone makes any change in these